zwave_me_server/Dockerfile.aarch64

```diff
@@ -1,30 +1,60 @@
-FROM balenalib/rpi-raspbian
-
-WORKDIR /opt/z-way-server
-
-ENV DEBIAN_FRONTEND=noninteractive 
-
-RUN apt-get update && \
-    apt-get install -qqy --no-install-recommends \
-    ca-certificates curl \
-    wget procps gpg iproute2 openssh-client openssh-server sudo logrotate jq
-
-RUN wget -q -O - https://storage.z-wave.me/RaspbianInstall |bash
-RUN rm -f /opt/z-way-server/automation/storage/*
+# Use the base image for the armhf architecture
+FROM ghcr.io/home-assistant/aarch64-base-debian:bullseye
+
+ENV ZWAY_PACKAGE_URL=https://storage.z-wave.me/z-way-server/z-way-4.1.2-wb7-lws16_armhf.deb
+ENV ZBW_PACKAGE_URL=https://storage.z-wave.me/z-way-server/zbw-2.6_all.deb
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY rootfs/ /
+
+# Install necessary libraries and their dependencies
+RUN dpkg --add-architecture armhf && apt-get update && apt-get install -y \
+libarchive13:armhf \
+libxml2:armhf \
+libssl1.1:armhf \
+libz1:armhf \
+libcurl4:armhf \
```

Comment:
Most of it is not needed if we rely on the apt-get to install dependencies. Better not to manage them manually, but rely on the package system